crates/networking/discv5/src/eth2.rs

```diff
@@ -0,0 +1,76 @@
+use alloy_primitives::{B256, Bytes, aliases::B32, bytes, fixed_bytes};
+use alloy_rlp::{Decodable, Encodable};
+use ream_consensus::{constants::FAR_FUTURE_EPOCH, fork_data::ForkData};
+use ssz::{Decode, Encode};
+use ssz_derive::{Decode, Encode};
+pub const GENESIS_VALIDATORS_ROOT: B256 =
+    fixed_bytes!("0x0000000000000000000000000000000000000000000000000000000000000000");
+pub const ENR_ETH2_KEY: &str = "eth2";
```

Comment:
```suggestion
use alloy_primitives::{B256, Bytes, aliases::B32, bytes, fixed_bytes};
use alloy_rlp::{Decodable, Encodable};
use ream_consensus::{constants::FAR_FUTURE_EPOCH, fork_data::ForkData};
use ssz::{Decode, Encode};
use ssz_derive::{Decode, Encode};

pub const ENR_ETH2_KEY: &str = "eth2";
pub const GENESIS_VALIDATORS_ROOT: B256 =
    fixed_bytes!("0x0000000000000000000000000000000000000000000000000000000000000000");
```

spaces and please order constants